### DISCUSSION TITLE
text object for additional products in Ticket Editor

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretix/pretix#3871] [Ideas] [answered]
There is currently no option in the ticket editor to print additional products as a text object in the "main product", is there?

Our "main product" is an admission product and with the additional products the guests choose on which days they want to come (we have a 10-day event).
Individual admission products for the individual days would be impractical; it would be nicer if the list of additional products could be written on the ticket.
Thanks a lot!

**Comment by raphaelm** [comment, marked as answer]
There is, it is called "List of Add-ons"

**Reply by navlisilvan** [reply]
Ah, thanks a lot!
